api/maven-api-model/src/main/mdo/maven.mdo

```diff
@@ -572,15 +550,12 @@
           <name>dependencies</name>
           <version>3.0.0+</version>
           <description>
-            <![CDATA[
-            This element describes all of the dependencies associated with a
-            project.
+            This element describes all the dependencies associated with a project.
             These dependencies are used to construct a classpath for your
             project during the build process. They are automatically downloaded from the
             repositories defined in this project.
-            See <a href="https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html">the
-            dependency mechanism</a> for more information.
-            ]]>
+            See &lt;a href="https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html">the
```

Comment:
`@see`?

Reply:
Isn't `@see` to link to another javadoc element ?

Reply:
Ah, https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#see

Reply:
Fixed